docs/_docs/subscribe/connect_options.md

```diff
@@ -1,5 +1,5 @@
 ---
 title: Connect Options
-permalink: /docs/subscribe/#connect-options
+permalink: /docs/publish/#connect-options
```

Comment:
is this link correct? there's a mix of /docs/publish and /docs/subscribe in the /subscribe directory

Reply:
Great catch. Definitely a bug.